frontend/packages/dev-console/src/components/import/DeployImageForm.tsx

```diff
@@ -27,21 +27,16 @@ const DeployImageForm: React.FC<FormikProps<FormikValues> & DeployImageFormProps
     />
     <ResourceSection />
     <AdvancedSection values={values} />
-    <ButtonBar errorMessage={status && status.submitError} inProgress={isSubmitting}>
-      <ActionGroup className="pf-c-form">
-        <Button
-          type="submit"
-          variant={ButtonVariant.primary}
-          isDisabled={!dirty || !_.isEmpty(errors)}
-          data-test-id="deploy-image-form-submit-btn"
-        >
-          Create
-        </Button>
-        <Button type="button" variant={ButtonVariant.secondary} onClick={handleReset}>
-          Cancel
-        </Button>
-      </ActionGroup>
-    </ButtonBar>
+    <FormFooter
+      handleReset={handleReset}
+      errorMessage={status && status.submitError}
+      isSubmitting={isSubmitting}
+      submitLabel="Create"
+      sticky
+      dataTestId="deploy-image-form-actions"
```

Comment:
Prefer scoping the `Form` with a `data-test-id`. Since we will only have single submit and cancel buttons.
Same comments for other forms.

Reply:
Updated test to use form scope to find the action buttons.